### PR TITLE
Copy ome.api.* documentation to omero.api.* (4)

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/KLCaZX4X/19-copy-ome-api-documentation-to-omero-api

Follow-up of #4547, #4569 and #4578, this PR copies more of the `ome.api.*` Javadoc documentation into the slice classes.

To review this PR, either build the slice documentation via release-slice2html or use the published CI documentation or OMERO.docs* artifacts. Check the links are all working and the method parameters match.

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#303](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/303/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/303/consoleText) for more details.
Possible conflicts:
  - PR #4452 jburel 'Ice 3.6'
    - components/blitz/resources/omero/api/RawFileStore.ice
    - components/blitz/resources/omero/api/RawPixelsStore.ice

**Comment by jburel** [comment]
This PR conflicts with https://github.com/openmicroscopy/openmicroscopy/pull/4452
the fact that there are double-quote in the description could also create error when parsing the slice object. This is due to a bug in ice.

**Comment by jburel** [comment]
Due to a bug in string parsing in ice,  check for single quote in the documentation

**Comment by sbesson** [comment]
See https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/ICE=3.6,jdk=8_LATEST,label=octopus/javadoc/

File: components/blitz/resources/omero/api/JobHandle.ice [hunk 1]
```diff
@@ -18,21 +18,101 @@ module omero {
...
+
+                /**
+                 * Returns the current message for job. May be set during
+                 * processing.
+                 */
                 idempotent string jobMessage()  throws ServerError;
+
+                /**
+                 * Returns <code>true</code> if the {@link omero.model.Job} is
+                 * running, i.e. has an attached {@link omero.model.Process}.
```
Comment (ximenesuk) [inline components/blitz/resources/omero/api/JobHandle.ice]: The i.e. here has the effect of the index entry being:

`Returns true if the model::Job is running, i.e.`

although out with this PR a change to

```
     * Returns <code>true</code> if the {@link omero.model.Job} is
     * running. I.e. has an attached {@link omero.model.Process}.
```

would make for a neater index entry.

**Comment by ximenesuk** [comment]
https://github.com/openmicroscopy/openmicroscopy/pull/4605/commits/b7d90055d935275539209f57b34be3a3169c9fe3#diff-32cd3b746a391445c25fd6e8c5f25621R120 needs the return value wrapping in a code block.

File: components/blitz/resources/omero/api/RawPixelsStore.ice [hunk 2]
```diff
@@ -41,14 +41,17 @@ module omero {
...
+                 * @param bypassOriginalFile Whether or not to bypass checking
+                 *        for an original file to back the pixel buffer used
+                 *        by this service. If requests are predominantly
+                 *        <code>write-only</code> or involve the population of
+                 *        a brand new pixel buffer using <code>true</code>
+                 *        here is a safe optimization otherwise
+                 *        <code>false</code> is expected.
                  *
-                 * See "read-only caveat" under {@link RawPixelsStore}
+                 * See "Read-only caveat" under {@link RawPixelsStore}
```
Comment (ximenesuk) [inline components/blitz/resources/omero/api/RawPixelsStore.ice]: For some reason I am not seeing this change in the rendered docs. But regardless of that does this change need to be made elsewhere in this file? Line 270 onwards has a few occurrences.

File: components/blitz/resources/omero/api/RawPixelsStore.ice [hunk 3]
```diff
@@ -65,41 +68,49 @@ module omero {
...
                  **/
                 idempotent void prepare(omero::sys::LongList pixelsIds) throws ServerError;
 
                 /**
-                 * Retrieves the in memory size of a 2D image plane in this pixel store.
-                 * @return 2D image plane size in bytes (sizeX*sizeY*ByteWidth).
+                 * Retrieves the in memory size of a 2D image plane in this
+                 * pixel store.
+                 * @return 2D image plane size in bytes
+                 *            <code>sizeX*sizeY*ByteWidth</code>.
```
Comment (ximenesuk) [inline components/blitz/resources/omero/api/RawPixelsStore.ice]: Should `ByteWidth` be `byteWidth` or is it really used with an uppercase `B`?

**Comment by sbesson** [comment]
@ximenesuk: pushed 2 commits addressing most of the formatting issues.

**Comment by jburel** [comment]
@ximenesuk: happy with the last changes?

File: components/blitz/resources/omero/api/JobHandle.ice [hunk 4]
```diff
@@ -77,7 +77,7 @@ module omero {
 
                 /**
                  * Returns <code>true</code> if the {@link omero.model.Job} is
-                 * running, i.e. has an attached {@link omero.model.Process}.
+                 * running, I.e. has an attached process.
```
Comment (ximenesuk) [inline components/blitz/resources/omero/api/JobHandle.ice]: If the comma after 'running' stays then it should be 'i.e.' (i.e. lower case I)

**Comment by ximenesuk** [comment]
@jburel thanks for the reminder. Other than the very tiny chnage to achieve perfect this looks okay. I'll leave @sbesson to fix that and then it can be merged.

**Comment by sbesson** [comment]
@ximenesuk/@jburel: I have a local branch to tackle the three last classes of this epic work - see https://trello.com/c/JQ9AcNa2/77-copy-ome-api-documentation-to-omero-api. Worth fixing it on the last PR?

**Comment by jburel** [comment]
Let's merge this PR, @ximenesuk minor edit can be done in the next PR.

**Pull request merged**